feedback.md

```diff
@@ -9,25 +14,42 @@ What did you need to configure and setup to make OAuth installed and work?
 What areas of Rails app code did you need to create/write/modify in order to change logging in to use OAuth?
   - Routes, controllers, models, views (if this includes migrations, tests, etc, or any of the above files, that's okay)
 
-What was one edge case you tested? How did this differ from the nominal case?
-  - Anything reasonable, use best judgment
+What was one controller test you updated?
+  - Anything reasonable, use best judgment.
+  - The obvious answer is: The `index` and `show` actions for work require authorization for Wave 3
 
 Why did we need to mock OAuth for testing?
   - To avoid using the GitHub as a dependency
   - Anything concrete about reducing risk around using actual GitHub accounts, resources, reducing complexity, etc. is fine too
 
 NOTES ABOUT TESTS:
 At project time, we give them:
-44 tests, 140 assertions, 0 failures, 0 errors, 4 skips
+61 tests, 140 assertions, 0 failures, 0 errors, 21 skips
 At project submission, we expect more tests/assertions, and 0 skips
 
  -->
 
-Feature | Feedback
----     | ---
-**General** |
-Completed OAuth | 
-Appropriate Tests on WorksController | 
-Appropriate Tests on UsersController | 
-Basic Authorization (who can see what) | 
-**Overall** |
+| Criteria | yes/no, and optionally any details/lines of code to reference |
+| --- | --- |
+| There are tests around the `User` model | ✔️?
+| There are tests for `UsersController#login` and `UsersController#logout` that use OAuth mocks | ✔️?
+| There are tests for `WorksController#upvote` that use OAuth mocks | ✔️?
```

Comment:
Maybe add something about tests for logged in and guest users in the works controller.